mysql-test/suite/rpl/r/rpl_show_slave_hosts.result

```diff
@@ -129,13 +129,114 @@ Server_id	Host	Port	Master_id	Gtid_State_Sent	Gtid_State_Ack
 show status like "Slaves_connected";
 Variable_name	Value
 Slaves_connected	2
+# Test 6.1: Check rpl_semi_sync_master_timeout=0 aka async
+# Note 1:   For asynchronous replication `Gtid_state_sent` is not null,
+#           but `Gtid_state_ack` is null.
+# Note 2:   We want that `rpl_semi_sync_master_timeout=0` be edge case,
+#           by mimicking asynchronous replication, for which
+#           `Gtid_state_ack` should not update (see Note 3).
+# Note 3:   With this option we want `Gtid_state_ack` to reflect
+#           the latest received transaction by the replica[s].
+select @@global.rpl_semi_sync_master_timeout;
+@@global.rpl_semi_sync_master_timeout
+10000
+set global rpl_semi_sync_master_timeout=0;
+select @@global.rpl_semi_sync_master_timeout;
+@@global.rpl_semi_sync_master_timeout
+0
+show variables like 'rpl_semi_sync_master_enabled';
+Variable_name	Value
+rpl_semi_sync_master_enabled	ON
+SHOW REPLICA HOSTS;
+Server_id	Host	Port	Master_id	Gtid_State_Sent	Gtid_State_Ack
+3	slave2	SLAVE_PORT	1	0-1-4	0-1-4
+2	localhost	SLAVE_PORT	1	0-1-4	0-1-4
+INSERT INTO t VALUES (505);
+show status like 'Rpl_semi_sync_master_yes_tx';
+Variable_name	Value
+Rpl_semi_sync_master_yes_tx	4
+# Q1: Here we need to get the `Gtid_State_Sent=0-1-5 != Gtid_State_Ack=0-1-4` for server_id=2 OR MAYBE THIS IS GOOD,
```

Comment:
With the IO thread ON (and no reason to pause/error) then the replica will `ACK` for the latest transaction, so both `.._sent` and `.._ack` would be `0-1-5`. So the current result seems right to me.